Chatto/Source/ChatController/ChatViewController.swift

```diff
@@ -156,7 +158,15 @@ public class ChatViewController: UIViewController, UICollectionViewDataSource, U
     }
 
     public func hideInputContainer(hide: Bool) {
-        self.heightConstraint.active = hide
+        if (hide) {
+            self.inputContainer.removeConstraint(topConstraint)
+            self.inputContainer.addConstraint(heightConstraint)
+        }
+        else {
+            self.inputContainer.removeConstraint(heightConstraint)
+            self.inputContainer.addConstraint(topConstraint)
```

Comment:
As opposed to adding/removing, you had used active before. Any reason for the change?